python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -240,6 +240,15 @@ def deblend(mExposure, footprint, config):
     ymin = bbox.getMinY()
     centers = [np.array([peak.getIy()-ymin, peak.getIx()-xmin], dtype=int) for peak in footprint.peaks]
 
+    # Choose whether or not to use the improved spectral initialization
+    if config.setSpectra:
+        if config.maxSpectrumCutoff == -1:
```

Comment:
Would it be simpler here (and below) to say that anything `<= 0` turns this off?